tests/cypress/integration/actions_tasks_objects/case_15_group_features.js

```diff
@@ -0,0 +1,132 @@
+                        // expected rgba(224, 224, 224, 0.533) to include [ 224, 224, 224, index: 4, input: 'rgb(224, 224, 224)', groups: undefined ]
+                        expect($bColorObjectsSidebarStateItem).contain(fillChangeAppearance.match(/\d+, \d+, \d+/));
+                    });
+                }
+            });
+        });
+        it('With group button unite two shapes. They have corresponding colors.', () => {
+            cy.get('.cvat-group-control').click();
+            for (let shapeToGroup of ['#cvat_canvas_shape_1', '#cvat_canvas_shape_2']) {
+                cy.get(shapeToGroup).click();
+            }
+            cy.get('.cvat-group-control').click();
+            for (let groupedShape of ['#cvat_canvas_shape_1', '#cvat_canvas_shape_2']) {
+                cy.get(groupedShape).should('have.css', 'fill').then($fillGroupedShape => {
+                    // expected rgb(250, 50, 83) to not equal rgb(224, 224, 224)
+                    expect($fillGroupedShape).to.not.equal(fillChangeAppearance);
+                    // expected rgb(250, 50, 83) to equal rgb(250, 50, 83)
+                    expect($fillGroupedShape).to.be.equal($fillGroupedShape);
+                    fillGroupedShape = $fillGroupedShape;
+                });
+            }
+            for (let objectSideBarShape of ['#cvat-objects-sidebar-state-item-1', '#cvat-objects-sidebar-state-item-2']) {
+                cy.get(objectSideBarShape).should('have.css', 'background-color').then($bColorobjectSideBarShape => {
+                    // expected rgba(250, 50, 83, 0.533) to not include [ 224, 224, 224, index: 4, input: 'rgb(224, 224, 224)', groups: undefined ]
+                    expect($bColorobjectSideBarShape).to.not.contain(fillChangeAppearance.match(/\d+, \d+, \d+/));
+                    // expected rgba(250, 50, 83, 0.533) to include [ 250, 50, 83, index: 4, input: 'rgb(250, 50, 83)', groups: undefined ]
+                    expect($bColorobjectSideBarShape).to.be.contain(fillGroupedShape.match(/\d+, \d+, \d+/));
+                });
+            }
+        });
+        it('With group button unite two track. They have corresponding colors.', () => {
+            cy.get('.cvat-group-control').click();
+            for (let trackToGroup of ['#cvat_canvas_shape_3', '#cvat_canvas_shape_4']) {
+                cy.get(trackToGroup).click();
+            }
+            cy.get('.cvat-group-control').click();
+            for (let groupedTrack of ['#cvat_canvas_shape_3', '#cvat_canvas_shape_4']) {
+                cy.get(groupedTrack).should('have.css', 'fill').then($fillGroupedTrack => {
+                    // expected rgb(250, 50, 83) to not equal rgb(224, 224, 224)
+                    expect($fillGroupedTrack).to.not.equal(fillChangeAppearance);
+                    // expected rgb(250, 50, 83) to equal rgb(250, 50, 83)
+                    expect($fillGroupedTrack).to.be.equal($fillGroupedTrack);
```

Comment:
``expect($fillGroupedTrack).to.be.equal($fillGroupedTrack);``?

Reply:
As well as for ```$fill```